net/freeradius/src/opnsense/scripts/Freeradius/generate_certs.php

```diff
@@ -65,10 +65,33 @@
                     ));
                     $pem_content .= "\n";
                     $cert_pem_content .= $pem_content;
-                    // generate ca pem file
+                    // generate cert pem file
```

Comment:
it would be better to exclude this chance and work on it in a dedicated PR. don't want to merge (and ship) both at the same time

Reply:
I will hand in a second PR for this